src/components/Modals/AddContactModal.jsx

```diff
@@ -151,39 +163,65 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
               )
             }}
           />
-          <DialogActions sx={{ width: '100%' }}>
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: isSmallScreen ? 'column' : 'row',
-                gap: isSmallScreen ? '10px' : '8px',
-                justifyContent: 'center',
-                alignItems: 'center',
-                width: '100%'
-              }}
+          <TextField
```

Comment:
The new additions to the modal are likely the cause for the failing tests as shown in the the test checks.